commands/questions/create-questions.js

```diff
@@ -1,51 +1,53 @@
-// Import the required modules
+/* eslint-disable no-console */
 const chalk = require('chalk');
 const figlet = require('figlet');
 const { validFileName, validPath } = require('./validate.js');
 
 // Print a header for the application in the terminal
 console.log(
   chalk.yellow(
-    figlet.textSync('Ontario.ca Toolkit', { horizontalLayout: 'full' })
-  )
+    figlet.textSync('Ontario\nFrontend', { horizontalLayout: 'full' }),
+  ),
 );
 
 // Define the questions for the inquirer prompt
 const createQuestions = [
-  {
-    type: "input",
-    name: "projectName",
+  { // cursor problem
+    type: 'input',
+    name: 'projectName',
     message:
-      "Specify your project name for the package.json file (lowercase, hyphens, and underscores only):",
-    default: "my-jamstack-project",
+      'Specify your project name for the package.json file (lowercase, hyphens, and underscores only):\n',
+    default: 'my-jamstack-project',
     validate: (value) => validFileName(value),
   },
   {
-    type: "input",
-    name: "projectDescription",
-    message: "Specify a short description for your project:",
-    default: "New Ontario.ca Jamstack Toolkit project",
+    type: 'input',
+    name: 'projectDescription',
+    message: 'Specify a short description for your project:\n',
+    default: 'New Ontario.ca Jamstack Toolkit project',
   },
```

Comment:
We should be consistent in our renaming. It's great that you changed the banner, but we should change all of the spots :)

Reply:
I did this fix real quick